patches/h3-js+3.7.2.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/node_modules/h3-js/dist/browser/h3-js.es.js b/node_modules/h3-js/dist/browser/h3-js.es.js
```

Comment:
This comment https://github.com/CartoDB/carto-react/pull/417/files#diff-337ead04a20e03f578a3166d4928bcf79a396bfd9df9e750662aeb689a5d5aceR2